review_practice/react-app/src/components/PersonnelController/style.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+
+export const PersonnelController = styled.div`
+    && {
+        width: 20rem;
+    }
+`;
+
+export const PersonnelItem = styled.div`
```

Comment:
p3;

```suggestion
export const PersonnelItem = styled.li`
```

리스트 아이템이라면 `li` 사용이 어떨까요? sematic web을 권장합니다.